app/models/ext_management_system.rb

```diff
@@ -50,6 +50,26 @@ def self.supported_for_catalog?
     catalog_types.present?
   end
 
+  def self.supported_for_label_mapping?
+    label_mapping_target_classes.present?
+  end
+
+  def self.supported_classes_for_label_mapping
+    supported_subclasses.select(&:supported_for_label_mapping?)
+  end
+
+  def self.supported_label_mapping_prefixes
```

Comment:
Unless we're going to have an `unsupported_label_mapping_prefixes` we can probably just call this `self.label_mapping_prefixes` or `all_label_mapping_prefixes`

Reply:
Done 👍